t/rest_server/spec.yaml

```diff
@@ -14,11 +14,43 @@
 
 apiVersion: 'cli.abcxyz.dev/v1alpha1'
 kind: 'Template'
-
-desc:
-  'An example template for a simple HTTP/JSON REST server.'
+desc: 'An example template for a simple HTTP/JSON REST server.'
+inputs:
+  - name: 'automation_service_account'
+    desc: 'Automation Service Account (ex: [account_name]@[project_name].iam.gserviceaccount.com)'
+  - name: 'wif_provider'
+    desc: 'Workload Identity Federation Provider (ex: projects/[project_id]/locations/global/workloadIdentityPools/[WIF_pool_name]/providers/[provider_name])'
+  - name: 'ar_repository'
+    desc: 'Artifact Registry Repository (repo name in AR)'
```

Comment:
Could we provide an example name for the AR name? That will make it clear whether this is supposed to be some/long/path/my-images vs just my-images

Reply:
Added ci-images as an example.